package.json

```diff
@@ -5,11 +5,12 @@
   "scripts": {
     "postinstall": "lefthook install",
     "dev": "next dev",
-    "build:check": "lefthook run pre-build",
```

Comment:
this will break window support because `&&` is bash

you should add this command as a step to lefthook `pre-build` step

Reply:
https://github.com/golemcloud/docs/pull/90/files#diff-ad6a01e589b8b1b214ca310dbb8d2e4314f6c612b921050c73c97455de43884dR24

under this step